source/externals/ptl/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ endif()
 
 # Check if project is being used directly or via add_subdirectory
 set(PTL_MASTER_PROJECT ON)
-if(NOT CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
+if(NOT CMAKE_CURRENT_SOURCE_DIR STREQUAL Geant4_SOURCE_DIR)
```

Comment:
PTL shouldn't be referencing Geant4

Reply:
Again, this is a case where `CMAKE_CURRENT_SOURCE_DIR` is needed, not a blind copy-paste